pkg/sql/pgwire/server.go

```diff
@@ -107,6 +107,7 @@ const (
 	version30     = 196608
 	versionSSL    = 80877103
 	versionCancel = 80877102
+	versionGSSENC = 80877104
```

Comment:
nit: I think future spelunkers would appreciate a comment above both this new constant and the whole block of constants explaining what they're for, how they're derived, and why you might want to add a new one.

Reply:
Added comments about this.